README.md

```diff
@@ -18,8 +18,8 @@ As users interact with the site, only the necessary parts of the site load on-de
 npm create qwik@latest
 # or
 pnpm create qwik@latest
-# or
 yarn create qwik@latest
+bunx create-qwik@latest
```

Comment:
Bun is still under development and doesn't support windows native build. 
Tomorrow, someone could raise or open a PR and say hey let's remove it from the docs. 🙂